Bio/Blast/NCBIXML.py

```diff
@@ -252,53 +252,53 @@ def _end_BlastOutput_version(self):
                 self._header.date = parts[2]
 
     def _end_BlastOutput_reference(self):
-        """A reference to the article describing the algorithm (PRIVATE).
+        """Yield the article describing the algorithm (PRIVATE).
```

Comment:
Yield implies a generator/iterator. Return seems wrong here - the function as no return value (Python default is just ``None``). How about:

```python
"""Record any article reference describing the algorithm (PRIVATE)."""
```